src/bin/nydus-image/main.rs

```diff
@@ -95,11 +104,19 @@ impl OutputSerializer {
                 .with_context(|| format!("can not open output file {}", f.display()))?;
             let trace = root_tracer!().dump_summary_map().unwrap_or_default();
             let version = format!("{}-{}", build_info.package_ver, build_info.git_commit);
+            let out_fs_version: u32 = match fs_version {
```

Comment:
Let's define `fs_version: RafsVersion` and use `RafsVersion::to_string()` here.

Reply:
Let's define `compressor: compress::Algorithm` and use `compress::Algorithm::to_string()` here.

Reply:
Or implement JSON Serialize for `RafsVersion` and `compress::Algorithm`.

Reply:
we can use `to_string` method